specs/DefaultDownloadDialog.md

```diff
@@ -0,0 +1,208 @@
+    }
+    else if (position == "Bottom-right")
+    {
+        newPosition =
+            CoreWebView2DefaultDownloadDialogPosition.BottomRight;
+        offset = new System.Drawing.Point(-padding, -padding);
+    }
+    webView.CoreWebView2.SetDefaultDownloadDialogPosition(
+        newPosition, offset);
+}
+```
+
+# API Details
+
+
+```c#
+/// The default download dialog can be aligned to any of the WebView corners.
+/// Use `SetDefaultDownloadDialogPosition` to specify a position and optional
+/// offset from that position.
+typedef enum COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION {
+
+  /// Top-left corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_TOP_LEFT,
+
+  /// Top-right corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_TOP_RIGHT,
+
+  /// Bottom-left corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_BOTTOM_LEFT,
+
+  /// Bottom-right corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_BOTTOM_RIGHT,
+
+} COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION;
+
+[uuid(9139c04d-8f37-42ae-8b63-01940c34d22f), object, pointer_default(unique)]
+interface ICoreWebView2_6 : ICoreWebView2_5
+{
+  /// `TRUE` if the default download dialog is currently open.
+  [propget] HRESULT IsDefaultDownloadDialogOpen(
+      [out, retval] BOOL* isDefaultDownloadDialogOpen);
+
+  /// Open the default download dialog. If the dialog is opened before there
+  /// are recent downloads, the dialog shows all past downloads for the
+  /// current profile. Otherwise, the dialog shows only the recent downloads
+  /// with a "See more" button for past downloads.
+  ///
+  /// \snippet ViewComponent.cpp ToggleDefaultDownloadDialog
+  HRESULT OpenDefaultDownloadDialog();
+
+  /// Close the default download dialog.
+  HRESULT CloseDefaultDownloadDialog();
+
+  /// Set the default download dialog position relative to the WebView bounds.
+  /// The dialog can be positioned against any of the WebView corners (see
+  /// COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION). When the WebView or dialog
+  /// changes size, the dialog keeps its position relative to the corner.
+  ///
+  /// The offset is an (x, y) coordinate in physical pixels that applies to the
+  /// dialog corner nearest to the COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION
+  /// specified. Use (0, 0) if no offset should be applied.
+  ///
+  /// The default position is top-right corner with offset (-30, 10). The dialog
+  /// may become partially or completely outside of the WebView bounds if the
+  /// WebView is small enough. The height of the dialog starts at 128 pixels and
+  /// expands with each new download until it reaches a maximum height of 650
+  /// pixels. The dialog also expands to maximum height when the "See more"
+  /// button is clicked on. The width is always 360 pixels.
+  ///
+  /// \snippet ViewComponent.cpp SetDefaultDownloadDialogPosition
+  HRESULT SetDefaultDownloadDialogPosition(
```

Comment:
This might be nicer as two properties with getter/setter. Does this need to be a method so they are always set at the same time?

Reply:
If its something like the download button in the browser the download button may move and the properties may need to change. I think its fine to leave as is?